target/linux/ramips/dts/mt7621_tplink_er605-v2.dts

```diff
@@ -84,7 +84,7 @@
 
 &gmac1 {
 	status = "okay";
-	label = "eth0";
+	label = "wan";
```

Comment:
as discussed `wan` or `wan1` both would be ok from my PoV

Reply:
All should be updated now. Thanks!